### PR TITLE
UPDATED: Skeleton for Med List

### PR DESCRIPTION
**Pull request by jleopold28**
I fixed a ton of merge conflicts by just making this new branch.

### CODE REVIEW

File: app/med-list-slider/med-list-slider.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ion-item-sliding *ngIf="doseMedication">
```
Comment (geoffreygraddy) [inline app/med-list-slider/med-list-slider.component.html]: I don't see where you have options for sliding.  Recommend if you are not using the sliding functionality to change it to a regular list.

Comment (jleopold28) [inline app/med-list-slider/med-list-slider.component.html]: Yeah I am going to change it, I was just using the template from the dose page.

File: app/pages/new-dose-medication/new-dose-medication.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <dose-amigos-toolbar [title]="title"></dose-amigos-toolbar>
+</ion-header>
+
+<ion-content padding  class="new-dose-medication">
+
+    <form (ngSubmit)="onSubmit()"
+          #requestForm="ngForm">
+
+        <h3>
+            Add a new medication.
```
Comment (jleopold28) [inline app/pages/new-dose-medication/new-dose-medication.html]: This is a placeholder until i finish this page

File: app/med-list-create-component/med-list-create.component.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="med-list-create">
+    <button fab fab-bottom fab-right fab-fixed
+            secondary
+            class="med-list-create-fab"
+            [navPush]="onClickPage">
+        <ion-icon name="person-add"></ion-icon>
```
Comment (mckoon) [inline app/med-list-create-component/med-list-create.component.html]: I don't think there is a good med-specific add icon.

Just make this "add":

`<ion-icon name="add"></ion-icon>`

File: app/pages/new-dose-medication/new-dose-medication.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.ned-dose-medication{
```
Comment (mckoon) [inline app/pages/new-dose-medication/new-dose-medication.scss]: `ned` - typo?

File: app/med-list-create/med-list-create.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import {DoseMedication} from "../dose-medication/dose-medication";
+
+export class MedListCreate {
```
Comment (mckoon) [inline app/med-list-create/med-list-create.ts]: `MedListCreate` -- What is this object?

There is nothing on the server that will match this. We just have `Med`, which maps to client-side `DoseMedication`.
https://github.com/dose-amigos/dose-amigos/wiki/Data-Objects-we'll-need

What do you envision we will do with this?

Comment (jleopold28) [inline app/med-list-create/med-list-create.ts]: Yeah I removed this, this was wrong.

**Comment by jleopold28** [comment]
I have fixed all the npm errors. This can be merged for now and then we can create new tickets to finish the med list screen.

**Comment by mckoon** [comment]
Seems like a good enough start to me.

+1

**Comment by geoffreygraddy** [comment]
:+1:

**Pull request merged**